packages/nc-gui-v2/components/smartsheet-column/EditOrAdd.vue

```diff
@@ -28,7 +30,7 @@ const columnToValidate = [UITypes.Email, UITypes.URL, UITypes.PhoneNumber]
 const uiTypesOptions = computed<typeof uiTypes>(() => {
   return [
     ...uiTypes.filter((t) => !isEdit || !t.virtual),
-    ...(!isEdit && meta?.value?.columns?.every((c) => !c.pk)
+    ...(!isEdit?.value && meta?.value?.columns?.every((c) => !c.pk)
```

Comment:
can `isEdit`really be undefined? Otherwise don't add optional chain operator `?` - it just makes things more confusing as it suggests "this value could be undefined".